nasa/osal
# Fix #339, Remove classic build

**Describe the contribution**
Remove classic build

**Testing performed**
Steps taken to test the contribution:
1. make prep
2. make install
3. Verify build/run cfs

**System(s) tested on:**
 - Hardware
 - Ubuntu 18.04
 - CFE 6.6, osal 5.0.3


**Contributor Info**
Anh Van, NASA Goddard
